x-pack/plugins/security/common/login_state.ts

```diff
@@ -4,9 +4,21 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { LoginLayout } from '../../../common/licensing';
+import { LoginLayout } from './licensing';
+
+interface LoginSelector {
+  enabled: boolean;
+  providers: Array<{
+    type: string;
+    name: string;
+    options: { description?: string; order: number };
```

Comment:
nit: are we using this `order` property anywhere? I couldn't find any references to it.

Reply:
Yeah, it seems we don't need it - I refactored loginState a bit in the latest commit to return only what we need and use on the client side.